fmriprep/cli/run.py

```diff
@@ -25,6 +25,9 @@
 from .. import config
 
 
+EXITCODE: int = -1
```

Comment:
You check for `> 0` in several places. Should they be `!= 0` or should this be something >0, e.g. 255?

Reply:
-1 is just a placeholder, it should never be this value